File: src/main/java/org/hamster/config/ApplicationsNameConfig.java
package org.hamster.config;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

import java.util.Map;

@Configuration
@ConfigurationProperties(prefix = "example")
@Data
public class ApplicationsNameConfig {
    /**
     * Ключ - название БД сервиса
     * Значение - схема где хранятся view
     */
    private Map<String, String> services;
}

File: src/main/kotlin/org/hamster/config/ApplicationsNameConfig.kt
package org.hamster.config

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.context.annotation.Configuration

/**
 * Ключ - название БД сервиса
 * Значение - схема где хранятся view
 */
@Configuration
@ConfigurationProperties(prefix = "example")
data class ApplicationsNameConfig(val services: Map<String, String>)